### PR TITLE
admin login page

### PR DESCRIPTION
**Pull request by giranezafiacre**
#### What does this PR do?
Creating Admin Login Page

#### Description of Task to be completed?
-Create Login Route Page using React Route Dom
-Integrate Firebase to the project and use the firebase Authentication Module to validate the user
-Add validated Email, Password, and Submit Button fields
-Create a dummy Admin page and Redirect to the Admin page when login is successful
-Alert the User with a notifier of the status of the login event once the user clicks the submit button.

#### How should this be manually tested?
-download source code run 'npm i' and run npm run start and write 'http:localhost:8080' and click in 'Admin Login'

#### Any background context you want to provide?
N/A

#### What are the relevant trello board stories?
https://trello.com/c/FRdPnFV1/35-login-page

#### Screenshots (if appropriate)
PR

#### Questions:
N/A

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Nav from './components/Nav';
 
-const app = document.createElement('app');
-ReactDOM.render(<Nav />, app);
+
+ReactDOM.render(<Nav />, document.getElementById('app')|| document.createElement('div'));
```
Comment (maximeish) [inline src/index.js]: this is great work but is the OR statement here necessary ?

Comment (IRANKUND) [reply src/index.js]: for me the OR is ok this index.js was causing error in two 2 ways ,
now it can take both

**Review by IRANKUND — COMMENTED** [review]
for me you did it well to solve the index.js problem

File: __mocks__/fileMock.js [hunk 2]
```diff
@@ -1 +1 @@
-module.exports = 'test-file-stub';
+module.exports = 'test-file-stub';
```
Comment (maximeish) [inline __mocks__/fileMock.js]: there are still some es lint errors. @giranezafiacre  you can add a new line at the end of each file to remove the errors

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import "../css/app.css";
+
+const App = () => {
+  return (
+    <BrowserRouter>
+      <div>
+        <Nav />
+        <Route exact path='/'>
+          <h1>React Page </h1>
+        </Route>
```
Comment (maximeish) [inline src/components/App.js]: this is great work. just on small thing you can change is this route **/** as it going to conflict the one I did so I think it is better to remove it

**Review by maximeish — CHANGES_REQUESTED** [review]

File: src/css/app.css [hunk 4]
```diff
@@ -1,12 +1,17 @@
-*{
```
Comment (maximeish) [inline src/css/app.css]: this file we named it as **App.css** with capital A in order to avoid conflicts

File: src/components/App.js (hunk 3 above)
Comment (giranezafiacre) [reply src/components/App.js]: will not fail to run when I remove it @maximeish? how about we remain as it is and by the time of merging into develop we remove conflicts manually?

**Review by maximeish — APPROVED** [review]

**Comment by giranezafiacre** [comment]
[Login Page](https://trello.com/c/FRdPnFV1/35-login-page)

**Pull request closed without merging**